workflow/metrics/scripts/metrics/pcr.py

```diff
@@ -100,3 +100,39 @@ def cell_cycle(adata, output_type, batch_key, label_key, adata_raw, n_threads=1,
     #     score = np.nan
 
     return score
+
+
+def pcr_random(adata, output_type, batch_key, label_key, adata_raw, var_key='metrics_features', n_threads=1, **kwargs):
```

Comment:
see comments in equivalent morans_i functions
```suggestion
def pcr_random(adata, output_type, **kwargs):
```